PROJECT

```diff
@@ -3,9 +3,13 @@ domain: opentelemetry.io
 layout: go.kubebuilder.io/v2
 repo: github.com/open-telemetry/opentelemetry-operator
 resources:
-- group: core
+-
+  controller: true
+  group: core
   kind: OpenTelemetryCollector
+  path: k8s.io/api/core/v1alpha1
   version: v1alpha1
-version: 2
+version: "3"
 plugins:
-  go.operator-sdk.io/v2-alpha: {}
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {} 
```

Comment:
as recommended by: https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.4.0/#gov2-change-go-project-plugin-object-to-manifests-and-scorecard-objects